app/controllers/server.rb

```diff
@@ -1,9 +1,108 @@
+require_relative '../models/client'
+require_relative '../models/processor'
+require 'pry'
+
 module RushHour
   class Server < Sinatra::Base
-    not_found do
+    def not_found(error_type)
+      @error_message = error_type
       erb :error
     end
+
+    get "/" do
+      erb :dashboard
+    end
     
-    # Write the rest of your controller code here!
+    get "/sample_sender" do
+      erb :sample_sender
+    end
+
+    get "/sources" do
+      status 200
```

Comment:
no need to manually set a status 200 - it will be returned with an erb (since this is also a successful request)